src/sentry/static/sentry/app/components/contextData.jsx

```diff
@@ -208,16 +208,9 @@ class ContextData extends React.Component {
   };
 
   render() {
-    // XXX(dcramer): babel does not support this yet
-    // let {data, className, ...other} = this.props;
-    const data = this.props.data;
-    const className = this.props.className;
-    const other = {};
-    for (const key in this.props) {
-      if (key !== 'data' && key !== 'className') {
-        other[key] = this.props[key];
-      }
-    }
+    // We do not use preserveQuotes, but require it to be removed from ...other
+    // eslint-disable-next-line no-unused-vars
+    const {data, className, preserveQuotes, ...other} = this.props;
     other.className = 'val ' + (className || '');
```

Comment:
This is old code but can we pull in the `classnames` library and just inline this to `<pre>`

Reply:
will do